test/run.sh

```diff
@@ -101,13 +122,18 @@ function main() {
     trap cleanup INT EXIT SIGHUP SIGINT SIGTERM
   fi
 
-  # Unit tests are fast, run them all first
+  # Unit tests are fast, run them all first, and always run them
```

Comment:
I think they should still respect the flag, if these grow to a large number the output will be noisy, making it hard to focus on another test suite.